### DISCUSSION TITLE
Error wdio browserstack

### DISCUSSION THREAD

**Discussion by firmansyahrosid** [webdriverio/webdriverio#12086] [General]
Hi, I got this error while integrating wdio with browserstaack.

`ERROR @wdio/browserstack-service: Error in executing getUserCaps with args [object Object]: TypeError: Cannot read properties of undefined (reading 'capabilities')`

**Comment by christian-bromann** [comment]
cc @kamal-kaur04 @ccharnkij

**Reply by paciadawid** [reply]
same error started appearing to me

**Reply by firmansyahrosid** [reply]
any updates @christian-bromann @kamal-kaur04 @ccharnkij ?

**Reply by christian-bromann** [reply]
Maybe worth reaching out to BrowserStacks Support?

**Reply by ccharnkij** [reply]
@firmansyahrosid I can try taking a look. Do you have any sample project or anything I can use to replicate this?

**Reply by kamal-kaur04** [reply]
Hi @firmansyahrosid @paciadawid @ccharnkij , we have picked this up. It would be great if you can share sample project as mentioned by @ccharnkij. We can fast track this.

**Reply by ken4ward** [reply]
Is there any answer to this as the issue persists. The same experience.

**Reply by kamal-kaur04** [reply]
Hay, we've raised the PR for the fix. Please refer - https://github.com/webdriverio/webdriverio/pull/12204

**Reply by sriteja777** [reply]
Hi, this is fixed in [8.23.0](https://github.com/webdriverio/webdriverio/releases/tag/v8.32.0).

**Reply by firmansyahrosid** [reply]
Terimakasih guys 👍
